### ISSUE TITLE
Handle default statements in switch/cases

### ISSUE THREAD

**Issue by mastef** [HaxeFoundation/as3hx#273]
as3 code for example:
```actionscript
package {
    public class Issue {
        public function Issue() {
            var param1 = "j";
            switch(param1)
            {
              case "a":
                    return 1;
              case "b":
                    return 2;
              case "c":
                    return 3;
              case "d":
                    return 4;
              case "e":
                    return 5;
              case "f":
                    return 6;
              case "g":
                    return 7;
              case "h":
                    return 8;
              default:
              case "i":
                    return 9;
            }
        }
    }
}
```
expected result
```haxe
package;

public class Issue
{
    public function Issue()
    {
        var param1 : String = "j";
        switch (param1)
        {
            case "a":
                    return 1;
            case "b":
                    return 2;
            case "c":
                    return 3;
            case "d":
                    return 4;
            case "e":
                    return 5;
            case "f":
                    return 6;
            case "g":
                    return 7;
            case "h":
                    return 8;
            default:
                    return 9;
        }
    }
}
```
actual result
```haxe
package;

public class Issue
{
    public function Issue()
    {
        var param1 : String = "j";
        switch (param1)
        {
            case "a":
                    return 1;
            case "b":
                    return 2;
            case "c":
                    return 3;
            case "d":
                    return 4;
            case "e":
                    return 5;
            case "f":
                    return 6;
            case "g":
                    return 7;
            case "h":
                    return 8;
            case "i":
                    return 9;
            default:
        }
    }
}
```

**Comment by mastef** [comment]
After testing this, `case "i" | _:` might not be the optimal solution. `default:` should be better, since it will only trigger when no other cases matched - even if it's defined above the cases ( see https://try.haxe.org/#1c2F8 )

**Comment by mastef** [comment]
Thanks for the merge @SlavaRa 
For `Changes.MD` : 
"Fixed conversion of `default` keyword within switch statements. fixes #273"

Do you think this could be now pushed to `haxelib`?
